workflow/tests/base_results/results_bills.csv

```diff
@@ -0,0 +1,330 @@
+base-appliances-coal.xml,144.0,1240.57,0.0,1384.57,96.0,120.73,216.73,0.0,0.0,0.0,0.0,72.99,1674.29
+base-appliances-dehumidifier-ief-portable.xml,144.0,1021.02,0.0,1165.02,96.0,7.41,103.41,0.0,0.0,0.0,0.0,0.0,1268.43
+base-appliances-dehumidifier-ief-whole-home.xml,144.0,1023.5,0.0,1167.5,96.0,7.15,103.15,0.0,0.0,0.0,0.0,0.0,1270.65
+base-appliances-dehumidifier-multiple.xml,144.0,1016.46,0.0,1160.46,96.0,7.93,103.93,0.0,0.0,0.0,0.0,0.0,1264.39
+base-appliances-dehumidifier.xml,144.0,1019.39,0.0,1163.39,96.0,7.2,103.2,0.0,0.0,0.0,0.0,0.0,1266.5900000000001
+base-appliances-gas.xml,144.0,1240.57,0.0,1384.57,96.0,162.56,258.56,0.0,0.0,0.0,0.0,0.0,1643.1299999999999
+base-appliances-modified.xml,144.0,1381.83,0.0,1525.83,96.0,119.37,215.37,0.0,0.0,0.0,0.0,0.0,1741.1999999999998
+base-appliances-none.xml,144.0,1060.15,0.0,1204.15,96.0,136.84,232.84,0.0,0.0,0.0,0.0,0.0,1436.99
+base-appliances-oil-location-miami-fl.xml,144.0,1368.21,0.0,1512.21,0.0,0.0,0.0,117.27,0.0,0.0,0.0,0.0,1629.48
+base-appliances-oil.xml,144.0,1240.57,0.0,1384.57,96.0,120.73,216.73,122.36,0.0,0.0,0.0,0.0,1723.6599999999999
+base-appliances-propane-location-portland-or.xml,144.0,996.52,0.0,1140.52,96.0,115.13,211.13,0.0,143.19,0.0,0.0,0.0,1494.84
+base-appliances-propane.xml,144.0,1240.57,0.0,1384.57,96.0,120.73,216.73,0.0,130.32,0.0,0.0,0.0,1731.62
+base-appliances-wood.xml,144.0,1240.57,0.0,1384.57,96.0,120.73,216.73,0.0,0.0,72.99,0.0,0.0,1674.29
+base-atticroof-cathedral.xml,144.0,1338.86,0.0,1482.86,96.0,173.01,269.01,0.0,0.0,0.0,0.0,0.0,1751.87
+base-atticroof-conditioned.xml,144.0,1523.91,0.0,1667.91,96.0,168.09,264.09,0.0,0.0,0.0,0.0,0.0,1932.0
+base-atticroof-flat.xml,144.0,1315.47,0.0,1459.47,96.0,113.32,209.32,0.0,0.0,0.0,0.0,0.0,1668.79
+base-atticroof-radiant-barrier.xml,144.0,999.3,0.0,1143.3,96.0,26.25,122.25,0.0,0.0,0.0,0.0,0.0,1265.55
+base-atticroof-unvented-insulated-roof.xml,144.0,1325.69,0.0,1469.69,96.0,134.28,230.28,0.0,0.0,0.0,0.0,0.0,1699.97
+base-atticroof-vented.xml,144.0,1333.91,0.0,1477.91,96.0,124.47,220.47,0.0,0.0,0.0,0.0,0.0,1698.38
+base-battery.xml,144.0,1345.9,0.0,1489.9,96.0,124.91,220.91,0.0,0.0,0.0,0.0,0.0,1710.8100000000002
+base-bldgtype-multifamily-adjacent-to-multifamily-buffer-space.xml,144.0,952.57,0.0,1096.57,96.0,99.17,195.17,0.0,0.0,0.0,0.0,0.0,1291.74
+base-bldgtype-multifamily-adjacent-to-multiple.xml,144.0,957.21,0.0,1101.21,96.0,57.73,153.73,0.0,0.0,0.0,0.0,0.0,1254.94
+base-bldgtype-multifamily-adjacent-to-non-freezing-space.xml,144.0,964.18,0.0,1108.18,96.0,208.91,304.91,0.0,0.0,0.0,0.0,0.0,1413.0900000000001
+base-bldgtype-multifamily-adjacent-to-other-heated-space.xml,144.0,939.7,0.0,1083.7,96.0,11.43,107.43,0.0,0.0,0.0,0.0,0.0,1191.13
+base-bldgtype-multifamily-adjacent-to-other-housing-unit.xml,144.0,955.82,0.0,1099.82,96.0,9.33,105.33,0.0,0.0,0.0,0.0,0.0,1205.1499999999999
+base-bldgtype-multifamily-shared-boiler-chiller-baseboard.xml,,,,,,,,,,,,,
```

Comment:
I think this file isn't getting a result because it has DSE -- but it actually uses DSE of 1. Can you allow the utility bill calculation to occur if both heating & cooling DSEs are 1?